editor/editor_properties_array_dict.cpp

```diff
@@ -1347,7 +1347,13 @@ void EditorPropertyDictionary::update_property() {
 
 			Variant value;
 			object->get_by_property_name(slot.prop_name, value);
-			Variant::Type value_type = value.get_type();
+			Variant::Type value_type;
+
```

Comment:
```suggestion

			Variant::Type value_type;
```
Better like this IMO.